### ISSUE TITLE
Deprecated(aerial.on_attach) warning with pylsp

### ISSUE THREAD

**Issue by ZhaoZuohong** [AstroNvim/AstroNvim#1192]
### Checklist

- [X] Using a stable version of Neovim (i.e. not neovim nightly)
- [X] `:PackerSync`
- [X] `:AstroUpdate`
- [X] Restarted AstroNvim

### AstroNvim version

v2.4.2

### Neovim version (>= 0.8)

NVIM v0.8.0-1210-gd367ed9b2

### Operating system/version

Linux Mint 21

### Describe the bug

With pylsp as python lsp server, when opening a python file, a popup warning message shows up:

Deprecated(aerial.on_attach): you no longer need to call this function
This function will be removed on 2023-02-01

### Steps to Reproduce

1. Install pylsp with `:LspInstall pylsp`
2. Restart nvim and open a python file
3. The popup warning will show in the upper right corner.

### Expected behavior

No popup

### Screenshots

![image](https://user-images.githubusercontent.com/34163622/198266334-add4748a-4347-42eb-954b-feef90e569ef.png)

**Comment by mehalter** [comment]
Thanks for reporting. This has been resolved already in the latest nightly

**Comment by LintaoAmons** [comment]
How to resolve this? I'm still getting the error

**Comment by mehalter** [comment]
@LintaoAmons you will need to pull the latest `nightly` release of AstroNvim or switch to `stable` AstroNvim.

**Comment by gigo6000** [comment]
I had to change  the channel and the branch to `nightly` to get the latest commits, the `stable` channel doesn't have the updates yet:

```lua
    channel = "nightly", -- "stable" or "nightly"
    version = "latest", -- "latest", tag name, or regex search like "v1.*" to only do updates before v2 (STABLE ONLY)
    branch = "nightly", -- branch name (NIGHTLY ONLY)
```

**Comment by ZhaoZuohong** [comment]
I confirm that the problem is fixed in `2.4.3`.
